src/plugins/newsfeed/public/components/newsfeed_header_nav_button.tsx

```diff
@@ -44,18 +46,19 @@ export const NewsfeedContext = React.createContext({} as INewsfeedContext);
 export type NewsfeedApiFetchResult = Rx.Observable<void | FetchResult | null>;
 
 export interface Props {
+  coreStart: CoreStart;
   apiFetchResult: NewsfeedApiFetchResult;
 }
 
-export const NewsfeedNavButton = ({ apiFetchResult }: Props) => {
+export const NewsfeedNavButton = ({ coreStart, apiFetchResult }: Props) => {
   const [showBadge, setShowBadge] = useState<boolean>(false);
   const [flyoutVisible, setFlyoutVisible] = useState<boolean>(false);
   const [newsFetchResult, setNewsFetchResult] = useState<FetchResult | null | void>(null);
 
   useEffect(() => {
     function handleStatusChange(fetchResult: FetchResult | void | null) {
       if (fetchResult) {
-        setShowBadge(fetchResult.hasNew);
+        setShowBadge(fetchResult.hasNew || true);
```

Comment:
what's the intent here? `(fetchResult.hasNew || true)` will always be true

Reply:
no it was me testing; thanks for catching it.